be/src/runtime/exec_env.cpp

```diff
@@ -167,6 +169,31 @@ Status ExecEnv::_init(const std::vector<StorePath>& store_paths) {
     _io_dispatcher = new workgroup::IoDispatcher(std::move(io_dispatcher_thread_pool));
     _io_dispatcher->initialize(num_io_threads);
 
+    // for real time
+    std::unique_ptr<ThreadPool> real_time_driver_dispatcher_thread_pool;
```

Comment:
do we need create real_time_driver_dispatcher_thread_pool and real_time_io_dispatcher_thread_pool by default？

Reply:
Yes, we need to, and a system will only allow the creation of one